libraries/AP_Notify/PixRacerLED.cpp

```diff
@@ -0,0 +1,266 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "PixRacerLED.h"
+#include <AP_GPS/AP_GPS.h>
+#include "AP_Notify.h"
+
```

Comment:
I tried without using this class and the LED's just don't work correctly and are hard to read.  So created this class to handle the light show better.